lib/streamlit/elements/map.py

```diff
@@ -101,16 +120,70 @@ def map(
 
         Parameters
         ----------
-        data : pandas.DataFrame, pandas.Styler, pyarrow.Table, numpy.ndarray, pyspark.sql.DataFrame, snowflake.snowpark.dataframe.DataFrame, snowflake.snowpark.table.Table, Iterable, dict, or None
-            The data to be plotted. Must have two columns:
-
-            - latitude called 'lat', 'latitude', 'LAT', 'LATITUDE'
-            - longitude called 'lon', 'longitude', 'LON', 'LONGITUDE'.
+        data : pandas.DataFrame, pandas.Styler, pyarrow.Table, numpy.ndarray,
+            pyspark.sql.DataFrame, snowflake.snowpark.dataframe.DataFrame,
+            snowflake.snowpark.table.Table, Iterable, dict, or None
+            The data to be plotted.
+
+        latitude : str or None
+            The name of the column containing the latitude coordinates of
+            the datapoints in the chart. This argument can only be supplied
+            by keyword.
+
+            If None, the latitude data will come from any column named 'lat',
+            'latitude', 'LAT', or 'LATITUDE'.
+
+        longitude : str or None
+            The name of the column containing the latitude coordinates of
```

Comment:
Should be: `The name of the column containing the `longitude` coordinates of`

Reply:
Eagle eyes!

🙏